img/posts/segwit-savings.gnuplot

```diff
@@ -3,6 +3,7 @@
 set style line 1 lc rgb '#8b1a0e' pt 4 ps 0.25 lt 1 lw 2
 set style line 2 lc rgb '#5e9c36' pt 4 ps 0.25 lt 1 lw 2
 set style line 3 lc rgb '#0025ad' pt 4 ps 0.25 lt 1 lw 2
+set style line 4 lc rgb '#7e2f8e' pt 4 ps 0.25 lt 1 lw 2
```

Comment:
blue-purple isn't the clearest contrast (eg for people with certain color blindness). Could you be persuaded to change this to purple-yellow or something else with more contrast?